server/models/Group.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+import { Op, DataTypes, sequelize } from '../sequelize';
+
+const Group = sequelize.define(
+  'group',
+  {
+    id: {
+      type: DataTypes.UUID,
+      defaultValue: DataTypes.UUIDV4,
+      primaryKey: true,
+    },
+    teamId: {
+      type: DataTypes.UUID,
+      defaultValue: DataTypes.UUIDV4,
+    },
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+  },
+  {
```

Comment:
You can add custom getters here, perhaps we add one for `membershipCount` that does the count query and can be called from the presenter? I'm not sure there is a more efficient way of doing that, short of caching the count in a column.

Reply:
investigated the custom getter route, but that would be an `async` getter which seems like creates a bit of complexity in sequelize — I'm gonna try just passing in a number for `memberCount` to the presenter and sending that across